logins-sql/tests/helpers/mod.rs

```diff
@@ -0,0 +1,309 @@
+
+        // Let users know why this is happening even if `log` isn't enabled.
+        println!("Running `npm install` in `integration-test-helper` to ensure it's usable");
+
+        let mut child = Command::new("npm")
+            .args(&["install"])
+            .spawn()
+            .expect("Failed to spawn `npm install`! (This test currently requires `node`)");
+
+        child.wait()
+             .expect("Failed to install helper dependencies, can't run integration test");
+    });
+    // We should still be in the script dir from HELPER_SETUP's call_once.
+    info!("Running helper script with command \"{}\"", cmd);
+
+    // node_args = ["index.js", cmd, ...cmd_args] in JavaScript parlance.
+    let node_args: Vec<&str> = ["index.js", cmd]
+        .iter()
+        .chain(cmd_args.iter())
+        .cloned() // &&str -> &str
+        .collect();
+
+    let child = Command::new("node")
+        .args(&node_args)
+        // Grab stdout, but inherit stderr.
+        .stdout(process::Stdio::piped())
+        .stderr(process::Stdio::inherit())
+        .spawn()?;
+
+    let output = child.wait_with_output()?;
+    if !output.status.success() {
+        let exit_reason = output.status.code()
+            .map(|code| code.to_string())
+            .unwrap_or_else(|| "(process terminated by signal)".to_string());
+        // Print stdout in case something helpful was logged there, as well as the exit status
+        println!("Helper script exited with {}, it's stdout was:```\n{}\n```",
+                 exit_reason, String::from_utf8_lossy(&output.stdout));
+        bail!("Failed to run helper script");
+    }
+    // Note: from_utf8_lossy returns a Cow
+    let result = String::from_utf8_lossy(&output.stdout).to_string();
+    Ok(result)
+}
+
+// It's important that this doesn't implement Clone! (It destroys it's temporary fxaccount on drop)
+#[derive(Debug)]
+pub struct TestAccount {
+    pub email: String,
+    pub pass: String,
+    pub cfg: FxaConfig,
+}
+
+impl TestAccount {
+    fn new(email: String, pass: String, cfg: FxaConfig) -> FailureResult<Arc<TestAccount>> {
+        info!("Creating temporary fx account");
+        // `create` doesn't return anything we care about.
+        let auth_url = cfg.auth_url()?;
+        run_helper_command("create", &[&email, &pass, auth_url.as_str()])?;
+        Ok(Arc::new(TestAccount { email, pass, cfg }))
+    }
+
+    pub fn new_random() -> FailureResult<Arc<TestAccount>> {
+        use rand::{self, prelude::*};
+        let mut rng = thread_rng();
+        let name = format!("rust-login-sql-test--{}",
+            rng.sample_iter(&rand::distributions::Alphanumeric).take(5).collect::<String>());
+        // Just use the username for the password in case we need to clean these
+        // up easily later because of some issue.
+        let password = name.clone();
+        let email = format!("{}@restmail.net", name);
+        Self::new(email, password, FxaConfig::stable_dev()?)
```

Comment:
`stable` cannot pass the tests at this time for strange reasons, but `latest` should work, if we want to land this then we should change this for now